assets/js/theme/common/product-details.js

```diff
@@ -287,8 +317,7 @@ export default class ProductDetails extends ProductDetailsBase {
             const quantityMin = parseInt($input.data('quantityMin'), 10);
             const quantityMax = parseInt($input.data('quantityMax'), 10);
 
-            let qty = parseInt($input.val(), 10);
-
+            let qty = forms.areNumbersOnly($input.val()) ? parseInt($input.val(), 10) : quantityMin;
             // If action is incrementing
             if ($target.data('action') === 'inc') {
```

Comment:
Important: put this validation to forms object as well

Reply:
Do you mean `If action is incrementing` block, right?

Reply:
done